src/pretix/api/views/event.py

```diff
@@ -429,7 +430,7 @@ def get(self, request, *args, **kwargs):
         # Get all orders of customer which belong to this event
         order_list = (Order.objects.filter(Q(event=event)
                                            & (Q(customer=customer) | Q(
-            email__iexact=customer.email))).select_related('event').order_by('-datetime'))
+                                               email__iexact=customer.email))).select_related('event').order_by('-datetime'))
```

Comment:
Why a redundant whitespace here?

Reply:
Hi @hongquan, its was format to add more whitespace when I run black fix: black src/pretix/api/views/event.py